packages/frontend/src/pages/ImportModel.svelte

```diff
@@ -132,37 +133,36 @@ export function goToUpPage(): void {
               on:dragover|preventDefault="{() => (dragging = true)}"
               on:dragleave|preventDefault="{() => (dragging = false)}"
               class="w-full cursor-pointer flex-col px-4 py-8 border-2 border-dashed rounded flex justify-center items-center">
-              <Fa size="1.1x" class="cursor-pointer text-purple-400" icon="{faFileImport}" />
-              <span>Drag & Drop or <strong class="text-purple-400">Choose file</strong> to import</span>
-              <span class="text-gray-800 text-sm">Supported format: .guff, .bin</span>
+              <Fa size="1.1x" class="cursor-pointer text-[var(--pd-link)]" icon="{faFileImport}" />
+              <span>Drag & Drop or <strong class="text-[var(--pd-link)]">Choose file</strong> to import</span>
+              <span class="opacity-50 text-sm">Supported format: .guff, .bin</span>
             </button>
           {:else}
             <!-- showing path -->
-            <label for="path" class="w-full block mb-2 text-sm font-bold text-gray-400">Path</label>
-            <input
-              class="flex w-full flex-col grow p-2 outline-none text-sm bg-transparent rounded-sm text-gray-700 placeholder-gray-700 border-charcoal-100 border-b mr-2"
-              bind:value="{localModel.path}"
-              name="path"
-              aria-label="model path"
-              readonly="{true}" />
+            <label for="path" class="w-full block mb-2 text-sm font-bold text-[var(--pd-content-card-header-text)]"
```

Comment:
should it use text-[var(--pd-label-text)] ? The same for all labels. It is too light otherwise imo
![image](https://github.com/containers/podman-desktop-extension-ai-lab/assets/49404737/98b80f9e-9741-45ef-88e3-c31b11a0d4e6)

Reply:
Ops, there was an error while building and something got wrong. This is what i actually see
![image](https://github.com/containers/podman-desktop-extension-ai-lab/assets/49404737/6da964ce-a7a2-45c6-a95f-42c7a76c1252)